mod1/class.tx_mksearch_mod1_IndizeIndizes.php

```diff
@@ -286,6 +290,34 @@ private function handleTrigger($oIntIndexSrv, &$configurations)
 
         return $status;
     }
+
+    /**
+     * Handle reset being indexed command from request.
+     * Reset entries older then time given in parameter "beingIndexedOlderThan" in minutes.
+     *
+     * @param tx_mksearch_service_internal_Index $oIntIndexSrv
+     * @param tx_rnbase_configurations           $configurations
+     *
+     * @return string
+     */
+    private function handleResetBeingIndexed($oIntIndexSrv, &$configurations)
```

Comment:
Please rename $oIntIndexSrv to $indexSrv. $configurations needs not be a reference.

Reply:
name fixes, $configuration remove because no need of it